src/Microsoft.TestPlatform.Common/Utilities/FakesUtilities.cs

```diff
@@ -164,8 +185,33 @@ private static bool TryGetFakesDataCollectorConfigurator(out Func<IEnumerable<st
                 }
             }
 #endif
-            configurator = null;
-            return false;
+            return null;
+        }
+
+        private static Func<IEnumerable<string>, bool, DataCollectorSettings> TryGetFakesNewDataCollectorConfigurator()
+        {
+            try
+            {
+                Assembly assembly = Assembly.Load(FakesConfiguratorAssembly);
+
```

Comment:
nit: are the extra white lines necessary?

Reply:
Looks like that's fixed.